src/__tests__/runStoredQueryTests.ts

```diff
@@ -0,0 +1,134 @@
+import { readFileSync } from "fs"
+import fetch from "lib/apis/fetch"
+import chalk from "chalk"
+const { USER_ACCESS_TOKEN: accessToken, USER_ID: userID } = process.env
+
+if (!accessToken && !userID) {
+  console.error(`
+  Cannot run queries without a user access token and user ID. You can get one by 
+  - 1. Signing in on staging.artsy.net
+  - 2. Open a dev console
+  - 3. and get the values \`sd.CURRENT_USER.id\` and \`sd.CURRENT_USER.accessToken\`
+  - 4. you can add these to your .env as:
+
+       USER_ID=[id]
+       USER_ACCESS_TOKEN=[accessToken]
+
+    You can also grab these from Reaction's .env if you have that set up.
+  `)
+  process.exit(1)
+}
+
+enum Env {
+  Prod,
+  Staging,
+  Dev,
+}
+
+const url = getURL(Env.Prod)
+
+const saleArtworkID =
+  "georg-jensen-beaded-slash-kugle-sterling-silver-cutlery-60"
+
+// We want to be able to run queries that require non-null variables
+// this is a map of a query name to their hard-coded variables.
```

Comment:
Ah yes, so when we switch to a db for persisted queries we should also store example variables that we can use to test all these queries.

Reply:
https://artsyproduct.atlassian.net/browse/PLATFORM-571